MANUAL.md

```diff
@@ -11982,8 +11982,8 @@ Metadata is data about a file which isn't the contents of the file.
 Normally rclone only preserves the modification time and the content
 (MIME) type where possible.
 
-Rclone supports preserving all the available metadata on files (not
-directories) when using the `--metadata` or `-M` flag.
+Rclone supports preserving all the available metadata on files and
```

Comment:
MANUAL.md is auto generated so you'll need to edit the original which is `docs/content/docs.md`

Reply:
And I see you beat me to it :) So I've dropped this one.